src/renderer/components/AutoSaveIndicator.tsx

```diff
@@ -33,12 +34,13 @@ function getClassNameAndMessage(savingState: SavingState, errorMessage?: string)
 type Props = {
     id?: string;
     savingState: SavingState;
-    errorMessage?: string;
+    errorMessage?: React.ReactNode;
+    intl: IntlShape;
 };
 
-export default function AutoSaveIndicator(props: Props) {
+const AutoSaveIndicator: React.FC<Props> = (props: Props) => {
     const {savingState, errorMessage, ...rest} = props;
-    const {className, message} = getClassNameAndMessage(savingState, errorMessage);
+    const {className, message} = getClassNameAndMessage(props.intl, savingState, errorMessage);
```

Comment:
Can we get `intl` from `useIntl` instead of using `injectIntl`? As far as I know, `injectIntl` doesn't really have a purpose in functional components. I'd just use it for class components at the moment